inc/frontend/namespace.php

```diff
@@ -58,7 +58,7 @@ function output_script() {
  */
 function mangle_images( $content ) {
 	// Find images
-	$searcher = '#<img[^>]+(?:class=[\'"]([^\'"]+wp-image-(\d+)[^\'"]*)|data-gaussholder-id="(\d+)")[^>]+>#x';
+	$searcher = '#<img[^>]+(?:class=[\'"]([^\'"]?wp-image-(\d+)[^\'"]*)|data-gaussholder-id="(\d+)")[^>]+>#x';
```

Comment:
Whoops, of course. It worked with my scenario, so I didn't dare to test it 😅

Fixed now.